### PR TITLE
Multiple UI improvements

### PR DESCRIPTION
**Pull request by asaadmahmoodspin**
Hey guys have a look. Changes include stuff from tickets:
https://mattermost.atlassian.net/browse/PLT-2381
https://mattermost.atlassian.net/browse/PLT-2384
https://mattermost.atlassian.net/browse/PLT-2371
https://mattermost.atlassian.net/browse/PLT-2385
https://mattermost.atlassian.net/browse/PLT-2301

Not sure if I've included the perfectscrollbar in the correct place. doesn't look right, so let me know if we need to include it elsewhere.

### CODE REVIEW

**Comment by crspeller** [comment]
@asaadmahmoodspin I there a reason perfect scrollbar is included as a non npm dependency? There is an npm package: https://www.npmjs.com/package/perfect-scrollbar

Also you have some ESLint failures.

**Comment by asaadmahmoodspin** [comment]
Done, have a look. @crspeller

File: webapp/package.json [hunk 1]
```diff
@@ -18,7 +18,7 @@
     "marked": "mattermost/marked#cb85e5cc81bc7937dbb73c3c53d9532b1b97e3ca",
     "match-at": "0.1.0",
     "object-assign": "4.0.1",
-    "perfect-scrollbar": "0.6.10",
+    "perfect-scrollbar": "^0.6.10",
```
Comment (jwilander) [inline webapp/package.json]: Might want to remove the `^` so that we don't accidentally update perfectscrollbar when we don't want to

**Comment by asaadmahmoodspin** [comment]
Done.

File: webapp/package.json [hunk 2]
```diff
@@ -53,8 +53,8 @@
     "webpack": "webpack/webpack#master"
   },
   "scripts": {
-      "check": "eslint --ext \".jsx\" --ignore-pattern node_modules --quiet .",
-      "build": "webpack",
-      "run": "webpack --progress --watch"
+    "check": "eslint --ext \".jsx\" --ignore-pattern node_modules --quiet .",
```
Comment (crspeller) [inline webapp/package.json]: Unintended spacing change?

File: webapp/root.jsx [hunk 3]
```diff
@@ -2,6 +2,7 @@
 // See License.txt for license information.
 
 import $ from 'jquery';
+require('perfect-scrollbar/jquery')($);
```
Comment (crspeller) [inline webapp/root.jsx]: Just for my information, why is this imported like this?

File: webapp/sass/layout/_navigation.scss [hunk 4]
```diff
@@ -89,7 +89,7 @@
 
                 &.info-popover {
                     @include background-size(100% 100%);
-                    background: url('../images/info__icon.png');
+                    background-image: url('../images/info__icon.png');
```
Comment (crspeller) [inline webapp/sass/layout/_navigation.scss]: Should be `images/info__icon.png` just in case the paths change.

File: webapp/package.json (hunk 2 above)
Comment (asaadmahmoodspin) [inline webapp/package.json]: screen shot 2016-03-23 at 5 34 11 pm
I think the spacing is inline with the above.

File: webapp/sass/layout/_navigation.scss (hunk 4 above)
Comment (asaadmahmoodspin) [inline webapp/sass/layout/_navigation.scss]: That doesn't work, so I'll keep it as it is.

**Comment by asaadmahmoodspin** [comment]
Also merged the commits.

**Comment by crspeller** [comment]
+1

**Comment by jwilander** [comment]
+1

**Pull request merged**